### ISSUE TITLE
More servers login issue

### ISSUE THREAD

**Issue by JiriVanek** [G-Node/WinGIN#139]
With the new version of WinGIN, when one of the configured servers is unavailable, the application displays error (erŕor 503 Service Unavailable) and shutdown during app startup. even though other servers are up and running

**Comment by achilleas-k** [comment]
Before we close this issue, a small question about #140.

Currently if the public GIN server isn't available (for some reason) but a local instance is, it will still fail to start, right?  I'm thinking maybe we can just show warnings for unavailable configured servers, regardless of which kind they are.  A potential scenario is a lab environment with a local instance but no access to public GIN (or public GIN is down for maintenance).

**Comment by JiriVanek** [comment]
Yes, the Gnode Gin server is mandatory. Also the offline use of wingin is impossible. I am not sure how to handle state when in cofiguration are several servers but some of them are offline. Maybe i will just check online status of default server and if default server is offline or unavailable I will show user the form with option to switch default server. Could that be solution?

**Comment by achilleas-k** [comment]
Yeah, I like that idea.
